include/openssl/ec.h

```diff
@@ -113,15 +115,15 @@ const EC_METHOD *EC_GF2m_simple_method(void);
  *  \param   meth   EC_METHOD to use
  *  \return  newly created EC_GROUP object or NULL in case of an error.
  */
-EC_GROUP *EC_GROUP_new_ex(OPENSSL_CTX *libctx, const EC_METHOD *meth);
+DEPRECATEDIN_3_0(EC_GROUP *EC_GROUP_new_ex(OPENSSL_CTX *libctx, const EC_METHOD *meth))
```

Comment:
As said above - it does not make sense to immediately deprecate a new function so it should be converted to an internal function instead.

Reply:
tyvm @t8m ! I will now sanity check all of these, since that never even crossed my mind ...

Reply:
> it does not make sense to immediately deprecate a new function so it should be converted to an internal function instead

Done in 17468cdf115b92d47ecb29952ec3dbd14d5b7ebd. I had to rebase due to conflict but I didn't squash yet just to keep the progress.

I sanity checked the rest of the functions for good measure. They're all ancient, so this was the only one.

Thanks for catching it :)